<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TargetEditorDialog</class>
 <widget class="QDialog" name="TargetEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="delete_selected_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>420</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="active_all_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deactive_selected_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>260</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Deactivate selected</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selected_deactive_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>160</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose inactive</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_all_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>460</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove all</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>481</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="uniformItemSizes">
    <bool>false</bool>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selected_active_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>120</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose active</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deactive_all_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>360</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Deactivate all</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="active_selected_Button">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>220</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Activate selected</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
